core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairGroup.java

```diff
@@ -101,6 +119,19 @@ public boolean execute()
         return successful;
     }
 
+    private boolean shouldContinue()
+    {
+        for (TableRepairPolicy repairPolicy : myRepairPolicies)
```

Comment:
Nit:
I believe this could be a little cleaner with streams. Something like this:
`myRepairPolicies.stream().anyMatch(policy::shouldRun(myTableReference))`

Reply:
+1, but I did it as:
```
myRepairPolicies.stream().allMatch(repairPolicy -> repairPolicy.shouldRun(myTableReference));
```
As we need to make sure it should run according to all policies.